sdk/nodejs/glacier/index.ts

```diff
@@ -2,9 +2,5 @@
 // *** Do not edit by hand unless you're certain you know what you are doing! ***
 
 // Export members:
-export * from "./detector";
```

Comment:
We probably want to delete these files from the source tree as well (both here and and in Python and Go).  Because we no longer have an external "overlays" folder, this process is a little tedious. We probably want some sort of script that uses the resources.go file to understand what hand authored overlays and removes all other files before running `tfgen`)

Reply:
Done in 1644b43. This is probably worth investigating in `tfgen` at some point though.